msgraph-sdk-raptor-compiler-lib/identifiers.json

```
@@ -9,6 +9,12 @@
     "_value": "team",
     "channel": {
       "_value": "team_channel",
+      "chatMessage": {
+        "_value": "<team_channel_chatMessage>",

```

Please remove the angle brackets to match the other value placeholders.
```suggestion
        "_value": "team_channel_chatMessage",
```